Notify/Notify.Functions/Notify.Functions/NotifyFunctions/Destination/UpdateDestination.cs

```diff
@@ -59,8 +60,18 @@ public static class UpdateDestination
                 }
                 else
                 {
-                    document = await createNewDocument(data, log, collection);
-                    result = new CreatedResult("", document.ToJson());
+                    string type = Convert.ToString(data.location.type);
```

Comment:
Declare variable in the beginning of the method.

Reply:
I have done as you did in the 2 functions below with the double longitude and latitude.
If you still want me to change tell me.

Reply:
I don't understand how is that related.
Move this variable declaration up to the beginning of the method.